src/App.jsx

```diff
@@ -9,32 +9,66 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-          </li>
-        </ul>
-      </div>
+export const Tab = ({ tab, isSelected, onTabSelected }) => (
+  <li
+    className={`${isSelected(tab) ? 'is-active' : ''}`}
+    data-cy="Tab"
+  >
+    <a
+      href={`#${tab.id}`}
+      data-cy="TabLink"
+      onClick={() => {
+        if (!isSelected(tab)) {
+          onTabSelected(tab.id);
+        }
+      }}
```

Comment:
Create separate handler and use reference here instead of usign anonymous function